specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1399,15 +1660,174 @@
+        },
+        "privateUrl": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gateway URL for accessing the non-current API Revision."
+        },
+        "isOnline": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Indicates if API revision is the current api revision."
+        },
+        "isCurrent": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Indicates if API revision is accessible via the gateway."
+        }
+      }
+    },
+    "ApiRevisionInfoContract" : {
+        "properties": {
+          "sourceApiId": {
+            "type": "string",
+            "description": "Id of API to be used to create the revision from."
+          },
+          "ApiVersionName": {
+            "type": "string",
+            "description": "Version identifier for the new API Version."
+          },
+          "ApiVersionDescription": {
+            "type": "string",
+            "description": "Description of new API Version"
+          },
+          "ApiRevisionDescription": {
+            "type": "string",
+            "description": "Description of new API Revision."
+          }
+        }
+    },
+    "ApiVersionSetContract" : {
+      "type": "object",
+      "description": "An API Version Set contains the common configuration for a set of API Versions relating ",
+      "properties": {
+        "Id" : {
+              "type": "string",
+              "description": "Identifier for existing API Version Set. Omit this value to create a new Version Set."
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of API Version Set."
+        },
+        "versioningScheme": {
+          "type": "string",
+          "description": "An value that determines where the API Version identifer will be located in a HTTP request.",
+          "enum": [
+            "Segment", "Query","Header"
+          ]
+        },
+        "versionQueryName": {
+          "type": "string",
+          "description": "Name of query parameter that indicates the API Version if versioningScheme is set to `query`."
+        },
+        "versionHeaderName": {
+          "type": "string",
+          "description": "Name of HTTP header parameter that indicates the API Version if versioningScheme is set to `header`."
+        }
+      }
+    },
+    "ApiReleaseCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiReleaseContract"
+          },
+          "description": "Page values."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "Paged Api Revision list representation."
+    },
+    "ApiReleaseContract": {
+      "type": "object",
+      "description": "API Release details",
+      "properties": {
+        "id": {
```

Comment:
>id [](start = 9, length = 2)

having examples, would help to run the validation on these contract.. not sure if "id" comes out when we call via ARM.

Reply:
just add one example.. on GET should be sufficient for validation, which has all the properties

---
In reply to: [140124298](https://github.com/darrelmiller/azure-rest-api-specs/pull/1#discussion_r140124298) [](ancestors = 140124298)

Reply:
ok